.travis.yml

```diff
@@ -4,6 +4,11 @@ install:
   # Travis's version of docker-compose doesn't understand :cached, and it's not required anyway, so this removes it
   - (cd devstack; sed -i 's/:cached//g' ./docker-compose-host.yml)
   - (cd devstack; make dev.clone)
+  - (cd edx-platform;
+      git remote add mitodl https://github.com/mitodl/edx-platform.git;
+      git fetch mitodl;
+      git checkout mitodl/master
+    )
```

Comment:
j/w: Is it necessary to use the mitodl fork here? Highly unlikely, but this could _theoretically_ be used outside of MIT. Just seems a bit off to hard-code a dependency on our fork. If we do need that fork for whatever reason, could we at least make the organization configurable via environment variable et. al., and default it to 'mitodl'?

Reply:
There was a test failure due to some package change in edx-platform so I needed to pin it to something. I'm not sure it would make sense to make this configurable here since it only affects testing. If we needed to change it in the future we can edit this file directly